qa/acceptance-tests/src/test/java/io/camunda/it/client/RolesByGroupIntegrationTest.java

```diff
@@ -369,26 +368,47 @@ void shouldRejectUnassigningRoleFromGroupIfGroupIdIsEmpty() {
   }
 
   @Test
-  void shouldRejectUnassigningRoleFromGroupIfGroupDoesNotExist() {
+  void shouldUnassigningRoleFromGroupIfGroupDoesNotExist() {
     // given
     final var groupId = Strings.newRandomValidIdentityId();
 
-    // when / then
-    assertThatThrownBy(
+    camundaClient
+        .newAssignRoleToGroupCommand()
+        .roleId(EXISTING_ROLE_ID)
+        .groupId(groupId)
+        .send()
+        .join();
+
+    Awaitility.await("Group is assigned to the role")
+        .ignoreExceptionsInstanceOf(ProblemException.class)
+        .untilAsserted(
             () ->
-                camundaClient
-                    .newUnassignRoleFromGroupCommand()
-                    .roleId(EXISTING_ROLE_ID)
-                    .groupId(groupId)
-                    .send()
-                    .join())
-        .isInstanceOf(ProblemException.class)
-        .hasMessageContaining(
-            "Expected to remove an entity with ID '"
-                + groupId
-                + "' and type 'GROUP' from role with ID '"
-                + EXISTING_ROLE_ID
-                + "', but the entity doesn't exist.");
+                assertThat(
+                        searchRolesByGroupId(
```

Comment:
🔧 Can you use the camunda client instead of the http client here ? I think we have the command now